applications/GeoMechanicsApplication/tests/test_line_loads.py

```diff
@@ -1,8 +1,11 @@
 import os
 
+import KratosMultiphysics as Kratos
 import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.StructuralMechanicsApplication as KratosStructural
 import test_helper
 
+
```

Comment:
I'm not entirely sure. I believe that when you would run `black` to enforce the prescribed style as described by PEP-8, it would ensure to have two blank lines between the `import`s and the actual code. But I have removed the blank line as you suggested, since I'm not entirely sure about which style to follow.